python_modules/dagster/dagster/_daemon/asset_daemon.py

```diff
@@ -266,10 +274,11 @@ def _initialize_evaluation_id(
                     continue
                 raw_cursor = instigator_data.cursor
                 if raw_cursor:
-                    stored_evaluation_id = AssetDaemonCursor.from_serialized(
-                        raw_cursor,
-                        asset_graph,
-                    ).evaluation_id
+                    stored_evaluation_id = (
```

Comment:
`raw_cursor` here is different than `raw_cursor` above right? worth a naming pass i think

Reply:
yeah good point -- updated